src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt

```diff
@@ -44,8 +44,8 @@ private fun isReportAvailable(version: String, availableSinceVersion: String): B
             versionsNumbers[2] >= reporterVersionNumbers[2]
 }
 
-private fun createReportOutputDir(target: Project, extension: KtlintExtension): File {
-    val reportsDir = File(target.buildDir, "reports/ktlint")
+private fun createReportOutputDir(target: Project, extension: KtlintExtension, sourceSetName: String): File {
+    val reportsDir = File(target.buildDir, "reports")
```

Comment:
What is the reason for removing `ktlint` dir?

Reply:
To align with the behavior like, as you can see in the screenshot, Android Lint. There is a specific reason why the `ktlint` folder is necessary?

Reply:
My point is that usually this reports are grabbed by the project CI. It is much easier to just grab all files from subfolder in `reports` dir then grab by some mask that can fail after plugin update, because there is new naming strategy.

Reply:
This is also a breaking change and will require a major version bump on the next release.